src/lib/import-export/import/validators/sql-validator.ts

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+import { BackupContents } from '../types';
+import { Validator } from './Validator';
+
+export class SqlValidator implements Validator {
+	canHandle( fileList: string[] ): boolean {
+		return fileList.length === 1 && fileList[ 0 ].endsWith( '.sql' );
+	}
+
+	parseBackupContents( fileList: string[], extractionDirectory: string ): BackupContents {
+		const extractedBackup: BackupContents = {
+			extractionDirectory: extractionDirectory,
+			sqlFiles: [],
+			wpContent: {
+				uploads: [],
+				plugins: [],
+				themes: [],
+			},
```

Comment:
Do we need to define `wp-content` folders if this validator is only meant to be used for SQL files?

Reply:
Thanks for your comment! I did that, to keep the API the same between the validators and to keep a common interface, that we give to the importer.

Reply:
Ok, makes sense. I understand we'll keep `wpContent` as part of the interface, but could we remove the subfolders. E.g.:
```suggestion
			wpContent: {},
```

Reply:
I'm not fan of improperly handled types tbh, as one doesn't have the whole picture by just looking at it.
Instead, I updated the wpContent object to be enforced. 
Thanks for your comment!